packages/router/router.ts

```diff
@@ -193,7 +193,7 @@ export interface Router {
    * Get/create a fetcher for the given key
    * @param key
    */
-  getFetcher<TData = any>(key?: string): Fetcher<TData>;
+  getFetcher<TData = any>(key: string): Fetcher<TData>;
```

Comment:
This is a type definition bug - key is not optional in the implementations of `getFetcher`/`deleteFetcher`